src/components/PremisesSelector.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ChangeEventHandler, FormEventHandler } from 'react'
+
+type Props = {
+    onChange: (premises: string) => void
+}
+
+const PremisesSelector: React.FC<Props> = ({ onChange }) => {
+    const handleSubmit: ChangeEventHandler<HTMLSelectElement> = (event) => {
+        event.preventDefault()
+        const premises: string = (event.currentTarget.value)
```

Comment:
Do you have prettier set up to format automatically?
It should remove these brackets, for example.

Reply:
The formatting in this file is off, actually. It's using four space indentation instead of 2.

Reply:
I'm going to configure our repo to lint automatically when we commit.

Reply:
I didn't notice that it was disabled. I've enabled it now & fixed the formatting issues.

Reply:
Sorry about that.